src/server/pps/server/pipeline_controller.go

```diff
@@ -159,21 +260,6 @@ func (m *ppsMaster) step(pipeline string, timestamp time.Time) (retErr error) {
 		tracing.FinishAnySpan(span, "err", retErr)
 	}()
 
-	// Retrieve pipelineInfo from the spec repo
-	op, err := m.newPipelineOp(opCtx, pipeline)
-	if err != nil {
-		// Don't put the pipeline in a failing state if we're in the middle
-		// of activating auth, retry in a bit
-		if auth.IsErrNotAuthorized(err) || auth.IsErrNotSignedIn(err) {
```

Comment:
This check was moved here: https://github.com/pachyderm/pachyderm/pull/6927/files#diff-d5f802277550bae9f4934102a2776841987d414328368ada3d674b4cae116435R131